src/js/controller/model.js

```diff
@@ -35,9 +35,10 @@ const Model = function() {
     };
 
     this.persistQualityLevel = function(quality, levels) {
-        var currentLevel = levels[quality] || {};
-        var label = currentLevel.label;
-        this.set('qualityLabel', label);
+        const currentLevel = levels[quality] || {};
+        if (currentLevel.bitrate) {
```

Comment:
Let's make sure that levels from providers have valid bitrate values or none at all.